python/paddle/fluid/dygraph/nn.py

```diff
@@ -810,6 +831,7 @@ def __init__(self,
                  pool_padding=0,
                  global_pooling=False,
                  use_cudnn=True,
+                 use_mkldnn=True,
```

Comment:
Since we use `FLAGS_use_mkldnn` to global control, do you need to add `use_mkldnn` attribute in each API?

Reply:
As far as I understood our agreement, we didn't want to add use_mkldnn attribute in standard op API, thus we have decided to use Flags here. However for dygraph ops we wanted to be able to control every op whether it should be executed with oneDNN or not. Similarly these ops have use_cuda attribute. Am I wrong?

Reply:
@phlrain What's your opinion about it?

Reply:
> However for dygraph ops we wanted to be able to control every op whether it should be executed with oneDNN or not

@grygielski Discussed with @phlrain, we think `FLAGS_use_mkldnn` is enough for dygraph, we don't need to control every op whether it should be executed with oneDNN or not